### ISSUE TITLE
🌋 E|F ➾ Allow users to invoke arbitrary commands for underlying tools such as LIGO

### ISSUE THREAD

**Issue by jchenche** [ecadlabs/taqueria#1460]
### 🚥 Status

- [X] 🔮 Product/Feature Design
- [X] ✍🏻 Architecture/Implementation Planning
- [X] 🏗️ Under Construction
- [x] ⛳ Feature Complete
- [ ] ⛵ GTM / Release

### 📐 Feature Description

**Proposal to allow arbitrary commands and options:**

_**Motivation:**_ sometimes LIGO developers want to use a command that’s currently unavailable in our LIGO plugin, such as `ligo compile expression`, and sometimes they want to use a option of a command that is available but doesn’t take such option, like `ligo compile --protocol`. If we allow some plugins such as the LIGO one to have a way of allowing these, then they wouldn’t be blocked by us.

_**The Spec would look like the following:**_ for some plugins (let’s take LIGO), they’ll provide a task aptly named `ligo` that takes a `--command` flag. The value for the flag would be a full LIGO command. UI would look something like this:
`taq ligo --command "compile contract --protocol kathmandu"`

_**Extra benefit:**_ via Google Analytics, we may be able to see which “non-taq task” commands and options (e.g. `taq ligo --command "someCmd someOptions"`) are most frequently used and then we can decide if we want to create a dedicated taq task for it.

### ⚖️ Acceptance Criteria

✨ Features: 
- [ ] ... 
 
 📚 Docs: 
- [ ] ... 

### 🔭 Scope

🏎️ In Scope: 
- ... 
 
 🙅‍♂️ Out of Scope: 
- ... 
 
 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
